packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1397,6 +1399,7 @@ export default class TradeStore extends BaseStore {
             ...this.proposal_info,
             [contract_type]: getProposalInfo(this, response, obj_prev_contract_basis),
         };
+        this.validation_params[contract_type] = this.proposal_info[contract_type].validation_params;
```

Comment:
Currently, we got validation_params only for half of contracts.

Reply:
@kate-deriv  it's ok, if no validation_params, there won't be any. But as soon as API is ready, we gonna receive them for all the contracts we need, and we won't need to change FE code for that, it'll start working automatically as required :)